uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-assignment.controller.spec.js

```diff
@@ -34,6 +34,19 @@ define([
       angular.extend(scope, Mock);
     }));
 
+    describe('on init', function () {
+      var resolvedData = { foo: 'foo', bar: 'bar' };
```

Comment:
The 'bar' property is not used in tests. It either should be used or be deleted.